server/storage/mvcc/watchable_store.go

```diff
@@ -482,6 +496,27 @@ func (s *watchableStore) progress(w *watcher) {
 	}
 }
 
+func (s *watchableStore) progress_all(force bool) {
```

Comment:
`force` seems to be unnecessary because it never becomes false.

Reply:
True. It is a non-obvious semantic difference between `progress` and `progress_all` though, so I thought it would be better to make it a bit more obvious.

Reply:
Sorry for my late response. The latest PR doesn't have the boolean flag and the parameter is a map of watchers, so it's clearer for me.
I think the semantics of plain `progress` is an independent topic. Focusing only one change in one PR might be good. Anyway, thanks for addressing my comment.